server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/GetMessagesMethodStepdefs.java

```diff
@@ -235,6 +235,11 @@ public void appendMessageWithSpecialCase(String messageName, String mailbox) thr
         appendMessage(messageName, "eml/htmlWithLongAndComplicatedContent.eml");
     }
 
+    @Given("^the user has a message \"([^\"]*)\" in \"([^\"]*)\" mailbox with iso charset")
+    public void appendMessageWithEncoding(String messageName, String mailbox) throws Throwable {
```

Comment:
For the record, and because we like explaning, when you are in main code it is useful to catch Exception instead of Throwable, so you don't catch very critical errors such as NoMoreMemoryException et MyProcessorIsBurningException (yesyes real Java errors).
For a declaration in a test class, it makes AFAIK no difference to declare methods as throwing Throwable or Exception. But our code style is to throw Exception for every test method, so please do.